### PR TITLE
Update the overview page

### PR DESCRIPTION
**Pull request by seakim-ms**
Added disclaimer for feedback collection

### CODE REVIEW

File: microsoft-365/admin/manage/servicenow-overview.md [hunk 1]
```diff
@@ -91,3 +91,6 @@ To understand each feature, see [Microsoft 365 support integration](https://stor
 
 > [!NOTE]
 > This app is not supported in regulated or restricted environments.
+
+> [!IMPORTANT]
+> The Microsoft 365 support integration app will occasionally prompt users for feedback about the app. If you don’t want users to be prompted for feedback, turn off this functionality in the app settings. For more information about Microsoft feedback policies, see [Learn about Microsoft feedback for your organization](https://docs.microsoft.com/en-us/microsoft-365/admin/misc/feedback-user-control?view=o365-worldwide). To change the feedback settings, follow the steps in the installation process.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/servicenow-overview.md]:
```suggestion
> The Microsoft 365 support integration app will occasionally prompt users for feedback about the app. If you don’t want users to be prompted for feedback, turn off this functionality in the app settings. For more information about Microsoft feedback policies, see [Learn about Microsoft feedback for your organization](/microsoft-365/admin/misc/feedback-user-control). To change the feedback settings, follow the steps in the installation process.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @seakim-ms, kindly commit the suggested change to change to a relative link. Many thanks.

**Comment by yogkumgit** [comment]
@seakim-ms Gentle reminder

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@seakim-ms Please implement the suggested changes, in the event of no response we may have to close this PR. Thanks

**Pull request merged**